Package@swift-6.swift

```diff
@@ -510,6 +511,19 @@ extension Target {
     )
   }
 
+  static var grpcHealthTests: Target {
+    .testTarget(
+      name: "GRPCHealthTests",
+      dependencies: [
+        .grpcHealth,
+        .grpcProtobuf,
+        .grpcInProcessTransport
+      ],
+      path: "Tests/Services/HealthTests",
+      swiftSettings: [._swiftLanguageMode(.v6), .enableUpcomingFeature("ExistentialAny")]
```

Comment:
Can you update the health module and health tests to use explicit imports like in Gus' PR https://github.com/grpc/grpc-swift/pull/2003?

I'm happy for this to be done in a separate PR as well (actually it probably needs to be done separately because Gus needs to do the imports on the code-gen too).